src/nodes.jl

```diff
@@ -194,7 +194,7 @@ macro op(ex)
         isa(arg_ex, Expr) && arg_ex.head === :parameters
     end
 
-    if param_idx > 0
+    if param_idx !== nothing
```

Comment:
Note that if `param_idx` is the result of a `find*` call, the result will only be `nothing` when not found on 0.7+. So for 0.6 support, you can do `something(param_idx, 0) > 0`. `something` selects the first non-`nothing` argument of its arguments and is available in Compat.